SingularityService/src/main/java/com/hubspot/singularity/resources/HistoryResource.java

```diff
@@ -215,6 +238,22 @@ public SingularityDeployHistory getDeploy(@ApiParam("Request ID for deploy") @Pa
   }
 
   @GET
+  @Path("/request/{requestId}/deploys/count")
```

Comment:
now that we are returning a paginated response with full data, not sure /count still makes sense as an endpoint name